src/tools/Makefile.am

```diff
@@ -28,21 +28,14 @@
 SUBDIRS += \
 	tools/prted \
 	tools/prun \
-	tools/pcc \
```

Comment:
I think we need to re-evaluate what `pcc` is used for, and whether it should actually be removed or not.

Reply:
I think we should not remove pcc, we just don't want to document it.

Reply:
Yeah, removal would be problematic in general. You could switch to "pmixcc", but then you'd need to modify all test and example compiles to point to the PMIx location and deal with those complexities (e.g., what is the user's PATH when they want to build an example).